docs/release.md

```diff
@@ -39,28 +39,45 @@ These artifacts are provided in their respective channels and their natural way
 of distribution.
 
 # Cadence and Timeline
-Both a new release branch and initial release candidate are cut on the **first
-business day of each month.**
-
-If no blocker issues are discovered the release candidate is **promoted to a full
-release after 5 business days.**
+New release branches are cut 3 times a year, so a release cycle will occupy about 15 weeks.
+This release cycle mirrors the Kubernetes' and allows KubeVirt to release after the community.
+This looks like:
+| Week Number in Year | K8s Release Number | K8s Release Week | KubeVirt Release Week |                     Note                    |
```

Comment:
Do we want to support the latest released Kubernetes version? It might be too optimistic to release on the same day.

Reply:
I can be more specific in the doc.  I was thinking 2-10 days should be enough for most KubeVirt release.